SR2024/2024-01-17-secondary-contact-info.md

```diff
@@ -0,0 +1,20 @@
+---
+to: Student Robotics 2024 secondary contacts, CC primary contacts
```

Comment:
Note: we failed to CC the primary contact in the end as this wasn't obviously possible using Gmail's mail merge (and also as we forgot).